### PR TITLE
chore: enable h2 for mobile

### PR DESCRIPTION
**Pull request by ktechmidas**
## Issue being fixed or feature implemented
http/2 needs to be enabled for mobile

## What was done?
enable http/2 for mobile to continue work with new TLS connections.

## How Has This Been Tested?
tested by @HashEngineering 

## Breaking Changes
None, this fixes things

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by QuantumExplorer — APPROVED** [review]

File: packages/dashmate/templates/platform/dapi/envoy/envoy.yaml.dot [hunk 1]
```diff
@@ -168,6 +168,7 @@ static_resources:
               "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.DownstreamTlsContext
               common_tls_context:
                 tls_certificates:
+                  alpn_protocols: [ "h2,http/1.1" ]
```
Comment (shumkov) [inline packages/dashmate/templates/platform/dapi/envoy/envoy.yaml.dot:171]: Do we need it only for self-signed? Look at the tempting.

**Review by shumkov — CHANGES_REQUESTED** [review]

File: packages/dashmate/templates/platform/dapi/envoy/envoy.yaml.dot (hunk 1 above)
Comment (ktechmidas) [reply packages/dashmate/templates/platform/dapi/envoy/envoy.yaml.dot:171]: It works for signed as well, Mobile needs it not DAPI.

Comment (ktechmidas) [reply packages/dashmate/templates/platform/dapi/envoy/envoy.yaml.dot:171]: I've already deployed this onto hp-masternode-29 and got Hash to check, it was working for him with the zerossl cert

Comment (shumkov) [reply packages/dashmate/templates/platform/dapi/envoy/envoy.yaml.dot:171]: No I mean there is another branch of if else condition where this setting is missed

Comment (HashEngineering) [reply packages/dashmate/templates/platform/dapi/envoy/envoy.yaml.dot:171]: For some reason the gRPC Java library requires that `h2` be made available via ALPN when using TLS/SSL communication with servers that have certificates.

**Comment by pshenmic** [comment]
Should go to master I suppose

**Comment by shumkov** [comment]
Closed in favor of #1102

**Pull request closed without merging**